src/datepicker/datepicker-navigation.ts

```diff
@@ -1,55 +1,99 @@
 import {Component, Input, Output, EventEmitter, ChangeDetectionStrategy} from '@angular/core';
-import {NavigationEvent} from './datepicker-view-model';
+import {NavigationEvent, MonthViewModel} from './datepicker-view-model';
 import {NgbDate} from './ngb-date';
 import {NgbDatepickerI18n} from './datepicker-i18n';
 import {NgbCalendar} from './ngb-calendar';
 
 @Component({
   selector: 'ngb-datepicker-navigation',
   changeDetection: ChangeDetectionStrategy.OnPush,
-  host: {'class': 'd-flex justify-content-between', '[class.collapsed]': '!showSelect'},
+  host: {'class': 'd-flex'},
   styles: [`
     :host {
       height: 2rem;
       line-height: 1.85rem;
     }
-    :host.collapsed {
-      margin-bottom: -2rem;
-    }
     .ngb-dp-navigation-chevron::before {
       border-style: solid;
       border-width: 0.2em 0.2em 0 0;
       content: '';
       display: inline-block;
+      width: 0.75em;
       height: 0.75em;
       transform: rotate(-135deg);
       -webkit-transform: rotate(-135deg);
       -ms-transform: rotate(-135deg);
-      width: 0.75em;
-      margin: 0 0 0 0.5rem;
     }
-    .ngb-dp-navigation-chevron.right:before {
+    .ngb-dp-arrow-btn.right {
+      float: right;
+    }
+    .right .ngb-dp-navigation-chevron:before {
       -webkit-transform: rotate(45deg);
       -ms-transform: rotate(45deg);
       transform: rotate(45deg);
-      margin: 0 0.5rem 0 0;
+    }
+    .btn-link {
+      cursor: pointer;
+      outline: 0;
+    }
+    .btn-link[disabled] {
+      cursor: not-allowed;
+      opacity: .65;
+    }
+    .ngb-dp-arrow {
+      flex-basis: auto;
+      flex-grow: 1;
+      padding-right: 0px;
+      padding-left: 0px;
+      margin: 0px;
+      width: 2rem;
+      height: 2rem;
+    }
+    .ngb-dp-arrow-btn {
```

Comment:
don't think you need `backgound-color` and `border-color` either, as `btn-link` should take care of it

Reply:
Hi
border-color is still needed and you are right about background-color not needed.
BR

Reply:
I think `.btn` already has transparent border

Reply:
Let's add btn ok

Reply:
Oh, you removed it, again the `btn btn-sm` were added by #1948 recently, so better to put it back for now. I think `btn btn-link` are supposed to be together